tests/max_config/nidaqmxMaxConfig.ini

```diff
@@ -195,6 +195,15 @@ BusType = PCIe
 PCI.BusNum = 0x0
 PCI.DevNum = 0x0
 
+[DAQmxDevice digitalPatternTester]
```

Comment:
"digitalPatternTester" sounds like https://www.ni.com/en/support/documentation/release-notes/product.ni-digital-pattern-driver.html

I'd prefer to call this "hsdioTester".